typedoc.json

```
@@ -0,0 +1,7 @@
+{
+    "pretty": true,
+    "includeVersion": true,
+    "disableSources": true,
+    "cleanOutputDir": true,
+    "theme": "markdown"
+  }
```

Add new line at end of file.

```suggestion
  }
  
```